civihr_employee_portal/templates/civihr-employee-portal-civihr-report-settings-age-group.tpl.php

```diff
@@ -6,7 +6,7 @@
         <span class="chr_search-result__total__count"><?php echo count($rows); ?></span>
       </div>
     </div>
-
+    <?php $classes .= ' table '; ?>
```

Comment:
leave a comment explaining why you are doing this. Also, isn't there a better place to do this other than a template?